source/PowerAdjusterView.mc

```diff
@@ -10,341 +10,340 @@ const a1 = 1.0899959d;
 const a2 = -0.0015119d;
 const a3 = 0.00000072674d;
 const rainbow = [
-     Gfx.COLOR_DK_BLUE,
-     Gfx.COLOR_DK_GREEN,
-     Gfx.COLOR_YELLOW,
-     Gfx.COLOR_ORANGE,
-     Gfx.COLOR_RED,
-     Gfx.COLOR_PURPLE ];
+    Gfx.COLOR_DK_BLUE,
```

Comment:
feel free to add the GREY color back, if you wish

Reply:
Added.